activity/obfuscate/README.md

```diff
@@ -0,0 +1,27 @@
+<!--
+title: Obfuscate Activity
+weight: 4618
+-->
+
+# Obfuscate
+This activity allows you to obfuscate the required value in the JSON using defined function.
+
+## Installation
+
+### Flogo CLI
+```bash
+flogo install github.com/microgateway/activity/obfuscate
+```
+
+## Configuration
+
+### Settings:
+| Name          | Type   | Description
+|:---           | :---   | :---     
+| operation     | string | The operation to perfirm (Allowed values are setLastFour) - **REQUIRED**
```

Comment:
I would add a section for "operations" and describe the operation.  Also fix the "perfirm" typo.   I would include an example of the what the operation does as well as a  description.